Scenario Based Samples/Iframe Full Server Embedding/README.md

```diff
@@ -0,0 +1,88 @@
+# Bold BI Full Server Based Embedding in ASP .NET Core MVC using IFrame
+
+This project was created using ASP.NET Core MVC. The application aims to demonstrate how to render Bold BI server with JWT in Iframe Embedding.
+
+## Bold BI Server Embedding view
+
+![Bold BI Server View](https://github.com/boldbi/samples/assets/129487075/37e0af6a-bd96-478c-ae91-5948840f257d)
+
+## Requirements/Prerequisites
+
+ * [Visual Studio Code](https://code.visualstudio.com/download)
+ * [.NET Framework](https://dotnet.microsoft.com/download/dotnet-core)
+
+ > **NOTE:** .NET 6.0 and 7.0 are supported
+
+#### Supported browsers
+  
+  * Google Chrome, Microsoft Edge, Mozilla Firefox and Safari.
+
+ ## Configuration
+ 
+ * Please configure the JWT Authentication in you UMS page based on your application Login and Logout URL. If it is not currently enabled, please refer to the following image or detailed [instructions](https://help.boldbi.com/multi-tenancy/site-administration/authentication/json-web-token/#steps-to-configure-jwt-in-bold-bi) to enable it. (ex: http://localhost:50000/ums/administration)
+ 
+    ![JWT Authentication](https://github.com/boldbi/samples/assets/129487075/0dc156e9-4d18-4ebf-b671-a72e045c70a6)
+
+ * As the application based url is localhost, we are setting the `Remote Login URL` and `Remote Logout URL` as below.
+ 
+    ![JWT Authentication](https://github.com/boldbi/samples/assets/129487075/97a6e08b-328a-42fb-a323-1b3784420960)
+    
+ * Copy the `Signing Key` from the JWT Authentication page and paste it into the `jwt:signingkey` value in the `appsettings.json` file.
+ 
+    ![appsettings.json](https://github.com/boldbi/samples/assets/129487075/37f439d6-3f58-43b3-807f-b90aeb3e2fbe)
+
+ * Open the UMS Site Settings page in `Bold BI Server`. Within `Authentication`, select the `General` tab, enable `Enable Default Authentication`, and `save` the changes.
+ 
+    ![Default Authentication](https://github.com/boldbi/samples/assets/129487075/9de1893d-52f0-40f7-b246-35169e97012a)
```

Comment:
Change the image with Marking Authentication section and JWT SSO selected as default authentication.

Reply:
changed the image